ml/backend/ggml/ggml.go

```diff
@@ -350,7 +359,6 @@ func (c Context) FromIntSlice(s []int32, shape ...int) (ml.Tensor, error) {
 
 func (c *Context) Close() {
 	if c != nil {
-		C.ggml_backend_sched_free(c.sched)
```

Comment:
This is freed when the context gets freed?

Reply:
After this change, the scheduler has the same lifetime as the model, rather than an individual context. In theory, it would be freed when the model gets freed but in practice we never do that and don't have an interface to do it other than killing the entire process